spec/support/vcr_cassettes/mhv_account_creation/should_not_create_an_account_if_one_already_exists.yml

```diff
@@ -0,0 +1,42 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: "<MHV_HOST>/mhv-api/patient/v1/account/register"
+    body:
+      encoding: UTF-8
+      string: '{"icn":"1012667122V019349","isPatient":true,"isPatientAdvocate":false,"isVeteran":true,"isChampVaBeneficiary":false,"isServiceMember":false,"isEmployee":false,"isHealthCareProvider":false,"isOther":false,"city":"Houston","country":"USA","zip":"77040","state":"Tx","address1":"20140624","email":"vets.gov.user+0@gmail.com","signInPartners":"VETS.GOV","termsVersion":"v3.2","termsAcceptedDate":"Tue,
+        09 May 2017 00:00:00 GMT"}'
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Apptoken:
+      - "<APP_TOKEN>"
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 400
+      message: Bad Request
+    headers:
+      Server:
+      - Apache-Coyote/1.1
+      Content-Type:
+      - application/json
+      Content-Length:
+      - '143'
+      Date:
+      - Fri, 19 May 2017 14:16:16 GMT
+      Connection:
+      - close
+    body:
+      encoding: UTF-8
+      string: '{"errorCode":150,"developerMessage":"Error:ICN: 1012667122V019349 is
```

Comment:
Actually I don't think this is critical, as this particular case is unlikely to ever occur. We would have the mhv_correlation_id in the MVI record. I just tested this scenario to see what would happen.